src/wallet/wallet.cpp

```diff
@@ -3868,11 +3840,10 @@ bool CWallet::ParameterInteraction()
     }
     nTxConfirmTarget = GetArg("-txconfirmtarget", DEFAULT_TX_CONFIRM_TARGET);
     bSpendZeroConfChange = GetBoolArg("-spendzeroconfchange", DEFAULT_SPEND_ZEROCONF_CHANGE);
-    fSendFreeTransactions = GetBoolArg("-sendfreetransactions", DEFAULT_SEND_FREE_TRANSACTIONS);
     fWalletRbf = GetBoolArg("-walletrbf", DEFAULT_WALLET_RBF);
 
-    if (fSendFreeTransactions && GetArg("-limitfreerelay", DEFAULT_LIMITFREERELAY) <= 0)
-        return InitError("Creation of free transactions with their relay disabled is not supported.");
+    if (GetBoolArg("-sendfreetransactions", false))
+        InitWarning("The argument -sendfreetransactions is no longer supported.");
```

Comment:
This was added in #9391 whose commit was included here, but then removed by the end of this PR.  Would have been cleaner to take it out of the first commit, but I'm going to aim for less history churn here, it doesn't hurt anything...